src/app/components/crypto-assets/choose-crypto-asset/fungible-token-asset-item.tsx

```diff
@@ -2,7 +2,7 @@ import { FlexProps } from 'leather-styles/jsx';
 
 import type { StacksFungibleTokenAssetBalance } from '@shared/models/crypto-asset-balance.model';
 
-import { StacksFungibleTokenAssetItem } from '@app/components/crypto-assets/stacks/fungible-token-asset/stacks-fungible-token-asset-item';
+import { StacksFungibleTokenAssetItemLayout } from '../stacks/fungible-token-asset/stacks-fungible-token-asset-item.layout';
```

Comment:
I think we could add a [convention](https://www.notion.so/trustmachines/Coding-conventions-8dbacf0398ca4b57a0ff823403c1cee7) to never use relative paths?